controls/1_1_filesystem_configuration.rb

```diff
@@ -20,7 +20,7 @@
 cis_level = input('cis_level')
 
 title '1.1 Filesystem Configuration'
-
+kernel_module
```

Comment:
this seems like a random paste, that should no be there

Reply:
Sorry about that, didn't catch this got polluted with changes for #146 but it's fixed now.